### PR TITLE
Product Catalog:

### PR DESCRIPTION
**Pull request by Mutalenic**
In this pull request we did the following:
Implemented startup actions by :

 - Present the user with a list of options to perform.
 - Let users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.
  
Created a main.rb file

### CODE REVIEW

File: classes/app.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+require_relative './item'
+
+class App
```
Comment (samiullahbahadur) [inline classes/app.rb:3]: Please change  App to startup

**Review by samiullahbahadur — COMMENTED** [review]
STATUS: Request changes
Hi @Mutalenic 
please change to App to startup  according to project requirement

File: classes/app.rb (hunk 1 above)
Comment (Mutalenic) [reply classes/app.rb]: done

**Review by samiullahbahadur — APPROVED** [review]
STATUS: Approve
Hi @Mutalenic 
you project is completed

**Pull request merged**